erpnext/accounts/report/accounts_receivable_summary/accounts_receivable_summary.py

```diff
@@ -53,7 +53,13 @@ def get_data(self, args):
 		)
 
 		if self.filters.show_gl_balance:
-			gl_balance_map = get_gl_balance(self.filters.report_date, self.filters.company)
+			if self.account_type == "Payable":
```

Comment:
@Jatin3128 
Move this if-else clause to `get_gl_balance` and pass 'account_type' as parameter.

Simple data types are always a better option for function / method parameters - `str` is better that `list(dict(list))`